Added 2 render conditions for custom entities (#2006)
MCreator/MCreator: plugins/mcreator-localization/help/default/entity/condition_is_body_visible.md

> If the returned value is true, the entity will be invisible. An example of this feature is players in the spectator mode.

Author: Oops, the worst is I tested the method several times. xD

If the returned value is true, the entity will be visible.

> When a player is in spectator mode, this value is false.

Reviewer: Если указанное здесь условие выполняется, то эта сущность будет видна.
Пока игрок в режиме наблюдателя, он не удовлетворяет этому условию.

While the returned value is true, the entity will be visible.

> While the player is in spectator mode, he does not meet this condition.

Reviewer: We're talking about an abstract player, so this would look better:
```suggestion
While a player is in spectator mode, they do not meet this condition.
```

The same about `fr_FR` if supported.
